recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ source:
   sha256: bd434d759948b4ef9d9e663b89f5019ea104afec6bf7a17267c624b0bfbc1a03  # [win]
 
 build:
-  number: 0
+  number: 1
   binary_relocation: false
   skip: true  # [osx or aarch64]
```

Comment:
Currently `linux_aarch64` is skipped. So this only affects `linux_ppc64le`

That said, the Arch Migrator PR ( https://github.com/conda-forge/cuda-nvprof-feedstock/pull/1 ) configured both Linux architectures for builds. So it makes sense to configure both for cross-compilation

That way it is easy to benefit from the cross-compilation changes should `linux_aarch64` be added here